### ISSUE TITLE
iPhone modal scrolling issue.

### ISSUE THREAD

**Issue by dkasper-was-taken** [cerner/terra-core#799]
# Issue Description
Scrolling issue on iphone with modal presented. 
I'm uncertain if it's an overlay issue, or some sort of z-index issue, but on the iPhone when attempting to scroll modal content sometimes the background content will instead of the modal content. Appears that both the background and the foreground at attempting catching scroll events.

## Issue Type
- [ ] New Feature
- [ ] Enhancement
- [x] Bug
- [ ] Other

## Environment
* Browser Name and Version: mobile Safari
* Operating System and version (desktop or mobile): iPhone

**Comment by jmsv6d** [comment]
After some investigation I have discovered that this issue is caused by momentum scrolling on iOS. On this platform any fixed position element that does not have overflow itself (that is actual overflow and not just overflow styling) will trigger momentum scrolling, causing the content behind it to scroll. I have found that this occurs in the popup, and everything that consumes it, as well as the date picker, and possibly more as it can happen in any fixed position element. I have looked at other ui libraries and have not found one that prevents this behavior. The best solution I have found is to add an event listener for touchmove and call event.preventDefault() unless the fixed container has overflow. This however, seems like a fragile solution, with the possibility of causing more problems than it's worth to fix this issue.
